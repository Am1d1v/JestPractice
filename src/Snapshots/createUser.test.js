import { createUser } from "./createUser"


describe('createUser', () => {

    it('Should create a user with firstname and lastname only', () => {

        const user = createUser({firstName: "D", lastName: 'Surname'});

        expect(user).toMatchSnapshot();

    })

})