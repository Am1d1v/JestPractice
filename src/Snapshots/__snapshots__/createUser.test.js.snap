// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createUser Should create a user with firstname and lastname only 1`] = `
{
  "email": "",
  "firstName": "D",
  "lastName": "Surname",
  "password": "password",
  "phone": "",
  "type": "USER",
}
`;
